pkgs/applications/virtualization/nvidia-docker/podman-config.toml

```diff
@@ -0,0 +1,13 @@
+disable-require = true
+#swarm-resource = "DOCKER_RESOURCE_GPU"
+
+[nvidia-container-cli]
+#root = "/run/nvidia/driver"
+#path = "/usr/bin/nvidia-container-cli"
+environment = []
+#debug = "/var/log/nvidia-container-runtime-hook.log"
+ldcache = "/tmp/ld.so.cache"
+load-kmods = true
+no-cgroups = true
+#user = "root:video"
+ldconfig = "@@glibcbin@/bin/ldconfig"
```

Comment:
Zwei @? Is this correct?

Reply:
It is. Let me dig around for reference material.

Reply:
The `@glibcbin@` is used for templating in that value using `substituteInPlace` while the first `@` indicates to `libnvidia-container` that the path to `ldconfig` is relative to the host (https://github.com/NVIDIA/libnvidia-container/blob/master/src/nvc_ldcache.c#L350-L359).